pkgs/development/tools/dapr/cli/default.nix

```
@@ -0,0 +1,32 @@
+{ buildGoModule, fetchFromGitHub, stdenv }:
+
+let
+  pname = "dapr";
+  version = "0.9.0";
+  sha256 = "1vdbh5pg3j7kqqqhhf4d9xfzbpqmjc4x373sk43pb05prg4w71s7";
+  vendorSha256 = "19qcpd5i60xmsr8m8mx16imm5falkqcgqpwpx3clfvqxjyflglpp";
+in buildGoModule {
+  pname = pname;
+  name = "${pname}-${version}";
+  version = version;
+
+  vendorSha256 = vendorSha256;
+
+  src = fetchFromGitHub {
+    owner = "dapr";
+    repo = "cli";
+    rev = "v${version}";
+    sha256 = sha256;
+  };
+
+  postInstall = ''
+    mv $out/bin/cli $out/bin/dapr
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/humio/cli";
+    description = "A CLI for managing and sending data to Humio";
+    license = licenses.asl20;
```

```suggestion
    license = licenses.mit;
```
https://github.com/dapr/cli/blob/master/LICENSE